src/errors.ts

```diff
@@ -138,6 +138,8 @@ export const enum Errors {
   InvalidNestedStatement,
   UnknownLabel,
   InvalidImportTail,
+  InvalidImportExpression,
+  InvalidImportBinding,
```

Comment:
Just noticed you forgot to add errorMessages for these two.

Reply:
Which causes:
```
> parseModule('import { foo } from "./foo.json" with { type: "json" };', { next: true })
/Users/huocp/playground/meriyah/src/errors.ts:176
        const message = '[' + line + ':' + column + ']: ' + exports.errorMessages[type].replace(/%(\d+)/g, (_, i) => params[i]);
                                                                                        ^

Uncaught TypeError: Cannot read properties of undefined (reading 'replace')
```

Reply:
Should it be named InvalidJSONImportBinding?

Reply:
I have made the changes that you have proposed, I have also deleted the InvalidImportExpression, since in the end it is not used after the changes to adapt to test252.